.husky/.gitignore

```diff
@@ -0,0 +1 @@
+_
```

Comment:
Don't think this is necessary with Husky v9.

Reply:
[cf9a424](https://github.com/electron/update-electron-app/pull/172/commits/cf9a424ff4ef21721907db3381471449328110a0)